openapi/resource-server.yaml

```diff
@@ -31,7 +31,7 @@ servers:
   - url: 'https://ilp.rafiki.money'
     description: 'Server for wallet address subresources (ie https://ilp.rafiki.money/alice)'
   - url: 'https://ilp.rafiki.money/.well-known/pay'
-    description: 'Server for when the wallet address has no pathname (ie https://ilp.rafiki.money)'
+    description: 'Server for when Payment Pointer has no pathname (ie https://ilp.rafiki.money)'
```

Comment:
is this an old version maybe?

Reply:
I think just need to remerge main in?

Reply:
good catch... lost it on a reverted merge. will fix

Reply:
fixed